src/components/web-chat/WebChat.stories.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios'
+import { WebChat } from './index'
+
+export default {
+  title: 'component/WebChat',
+  component: WebChat,
+  argTypes: {}
+}
+
+// set window parameters
+window['webChatUrl'] = 'https://testchatservices.maximusbc.ca/Home/Chat'
+window['webChatReason'] = 'SBC_Registries'
+window['webChatStatusUrl'] = 'https://mcap-schedule-3a0694-test.apps.silver.devops.gov.bc.ca/api/status/SBC_Registries'
```

Comment:
These are the current Dev parameters. These allow demoing this component in StoryBook.